dashboard/src/views/partials/LoginBox.vue

```diff
@@ -27,22 +27,20 @@
 			</div>
 		</div>
 		<div class="absolute bottom-4 z-[1] flex w-full justify-center">
-			<FrappeLogo class="h-4" />
+			<BrandLogo :footer="1" />
```

Comment:
not very explicit with using 1
```suggestion
			<BrandLogo :footer="true" />
```